avocado/utils/pmem.py

```diff
@@ -118,6 +124,69 @@ def run_daxctl_list(self, options=''):
         cmd = '%s list %s' % (self.daxctl, options)
         return json.loads(process.system_output(cmd))
 
+    def dax_online_memory(self, device, region='', no_movable=False):
```

Comment:
If there is no region == '' and region is optional, then `region=None`, would be better here.

Reply:
Good catch!